HiP-Achievements/Controllers/AchievementsController.cs

```diff
@@ -175,9 +176,8 @@ public IActionResult GetAchievementById(int id)
                                           .AsQueryable()
                                           .FilterByStatus(AchievementQueryStatus.Published);
 
-            var actions = _db.GetCollection<Action>(ResourceTypes.Action)
-                                          .AsQueryable()
-                                          .Where(x => x.UserId == User.Identity.GetUserIdentity());
+            var actions = await _userStoreService.Actions.GetAllActionsAsync();
+            var exhibitVisitedActions = actions.Items.Where(x => x.Type == "ExhibitVisited").ToList();
```

Comment:
The string that is used for comparison here should not be hard coded. We should be able to extract it from the UserStoreService somehow (maybe something like new ExhibitVisitedAction().TypeName could be required)

Reply:
My bad , these classes are not available in the SDK project. Than it's fine